fed/_private/fed_call_holder.py

```diff
@@ -35,7 +35,8 @@ def options(self, **options):
         self._options = options
         return self
 
-    def internal_remote(self, *args, **kwargs):
+    def internal_remote(self, invoking_frame=None, *args, **kwargs):
```

Comment:
Since Line 39 asserts invoking_frame should not be None, so no default value None to `invoking_frame` is better.